test.js

```diff
@@ -0,0 +1,45 @@
+/*
+
+assert that objects, predicates, teams, and {future stuff} are present.
+
+when writing the tests, it'd be good to assert all of the clauses that build.js claims to enforce
+
+contents are alphabetized, contents don't have dupes, etc.
```

Comment:
within the 'etc' category here would also be the "contents are lower-case ascii alphabet characters",  "no empty strings", etc.

Reply:
added more tests!